test/each_benchmark.cc

```diff
@@ -8,12 +8,42 @@
 
 int main(int argc, char **argv)
 {
-  // users may specify the number of entities they'd like to spawn from the
-  // command line
-  int numEntities = 100;
-  if (argc == 2)
+  // command line arguments are as follows:
+  //  * the number of entities to create (required)
+  //  * the number of entities that should have a component added/removed
+  //    in between Each calls (optional). If this argument is not specified,
+  //    no components will be added/removed from entities between Each calls
```

Comment:
The README should be updated to reflect the new argument order/requirements (make a note about how `LinearVelocity` is the component added/removed)

Reply:
https://github.com/adlarkin/simple_ECM/pull/2/commits/d9edbbedf895b7eb858610c007f0466e1b698037